googletest/include/gtest/internal/gtest-filepath.h

```diff
@@ -173,6 +165,9 @@ class GTEST_API_ FilePath {
   bool IsAbsolutePath() const;
 
  private:
+  FilePath(const FilePath& rhs) = default;
```

Comment:
A few questions here:
1. Why not allow users to copy FilePath? is it incorrect or we just believe it's inefficient?
2. Does the implementation need to make copies? If not, can we move these to the public section and make them `= delete`?